components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx

```diff
@@ -57,6 +57,12 @@ const EnterpriseEditionLeftPanel: React.FC<EnterpriseEditionProps> = ({
 
     return (
         <div className='EnterpriseEditionLeftPanel'>
+            {isTrialLicense && <div className='pre-title'>
+                <FormattedMessage
+                    id='admin.license.enterpriseEdition'
+                    defaultMessage='Enterprise Edition'
+                />
+            </div>}
```

Comment:
Why is this title only for trial licenses?

Reply:
According to the design it only appears in Start and when it is a Trial license. @annelieseklein can confirm that.